src/declarativewebcontainer.cpp

```diff
@@ -139,6 +135,10 @@ void DeclarativeWebContainer::setWebPage(DeclarativeWebPage *webPage)
         m_webPage = webPage;
 
         if (m_webPage) {
+            connect(m_webPage, SIGNAL(navigationHistoryChanged()), this, SIGNAL(canGoForwardChanged()));
+            connect(m_webPage, SIGNAL(navigationHistoryChanged()), this, SIGNAL(canGoBackChanged()));
+            connect(m_webPage, SIGNAL(urlChanged()), this, SIGNAL(urlChanged()));
+            connect(m_webPage, SIGNAL(titleChanged()), this, SIGNAL(titleChanged()));
```

Comment:
I'll rephrase, move all connections to one place :+1: 

For documentation reason I'd keep connections still unique.

Regarding forwarding signals, it's not a big deal. Just that there will be two connections now instead of using one connection for handling title and url changes.